libsigner/src/libsigner.rs

```diff
@@ -80,7 +81,13 @@ pub trait SignerMessage<T: MessageSlotID>: StacksMessageCodec {
 lazy_static! {
     /// The version string for the signer
     pub static ref VERSION_STRING: String = {
-        let pkg_version = option_env!("STACKS_NODE_VERSION").unwrap_or(env!("CARGO_PKG_VERSION"));
+        let pkg_version = option_env!("STACKS_NODE_VERSION").or(Some(STACKS_SIGNER_VERSION));
         version_string("stacks-signer", pkg_version)
     };
 }
+
+#[test]
+fn test_version_string() {
+    let version = VERSION_STRING.to_string();
+    assert!(version.contains(format!("stacks-signer {}", STACKS_SIGNER_VERSION).as_str()));
```

Comment:
The variable `version` is unnecessary here:

 ```suggestion
     assert!(VERSION_STRING.contains(&format!("stacks-signer {STACKS_SIGNER_VERSION}")));
```